### DISCUSSION TITLE
Display button functions on all screens

### DISCUSSION THREAD

**Discussion by JustinZz7** [OnionUI/Onion#1298] [Ideas]
To help with usability, persistently display the button functions on all screens
We display the legend for Select and Back, which arguably are the buttons that need legends the least, but X and Y are not labeled
![image](https://github.com/OnionUI/Onion/assets/93228546/e0d6fe87-0ef3-49a1-8708-af88e8665d96)

**Comment by schmurtzm** [comment]
It would require to change it dynamically when we change Xand Y affectation in Tweaks app. 
Probably require to edit all the themes too.
And ideally it would require to modify MainUI which is closed source....
So the idea is interesting but currently it's quite complex to achieve 😅
